### ISSUE TITLE
update style of buttons in brewery cards

### ISSUE THREAD

**Issue by cdrani** [JasonFritsche/BreweryFinder#83]
The brewery info cards currently look like this:

![Screen Shot 2019-04-20 at 13 21 56](https://user-images.githubusercontent.com/18746599/56461397-8bd4a780-636f-11e9-9f87-50252c25f61a.png)

I think it should be better suited that the buttons should be side by side as it would look better.

Changes to consider:

* [x] change 'Check Them Out' to 'Visit'
* [x] change 'View Map' to 'Map'
* [x] display buttons side by side
* [x] add some padding to buttons to beef up buttons
* [x] reduce the blur/box-shadow or whatever it around the buttons from bleeding out so much

Any additional changes you deem fit will also be accepted.

**Comment by chrisdesilva** [comment]
I'll take this one

**Comment by cdrani** [comment]
@chrisdesilva: Great work on this issue. I love how it looks now. I noticed this issue was not closed after your PR was merged. This can usually be done manually by closing the issue here on github or automatically if a key word and the issue number is mentioned either in the commit message body or PR description as mentioned [here](https://help.github.com/en/articles/closing-issues-using-keywords).

Closing this now. Thanks
